kornia/augmentation/_3d/mix/transplantation.py

```diff
@@ -0,0 +1,14 @@
+from kornia.augmentation._2d.mix.transplantation import RandomTransplantation
+from kornia.augmentation._3d.base import AugmentationBase3D
+
+__all__ = ["RandomTransplantation3D"]
+
+
+class RandomTransplantation3D(RandomTransplantation, AugmentationBase3D):  # type: ignore
+    """RandomTransplantation3D augmentation.
+
```

Comment:
BTW, do you have any 3D images by hand that we can show here?

Reply:
Unfortunately not because I never worked with 3D images so far. Some MRI or CT images of a 3D segmentation task would probably be good examples but the visualization is more difficult. But this is better suited for a separate PR IMHO.